packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -774,23 +787,22 @@ describe('SelectedNetworkController', () => {
 
     // TODO - improve these tests by using a full NetworkController and doing more robust behavioral testing
     describe('when the domain is a snap (starts with "npm:" or "local:")', () => {
-      it('returns a proxied globally selected networkClient and does not create a new proxy in the domainProxyMap', () => {
-        const { controller, domainProxyMap, messenger } = setup({
+      it('calls to NetworkController:getSelectedNetworkClient and creates a new proxy provider and block tracker with the proxied globally selected network client', () => {
+        const { controller, messenger } = setup({
           state: {
             domains: {},
           },
-          useRequestQueuePreference: true,
+          useRequestQueuePreference: false,
```

Comment:
why was this flipped to false?

Reply:
This is testing if per dapp selected network is turned off, which it really shouldn't ever be at this point. Going to be removing `useRequestQueuePreference` altogether soon

Reply:
nvm the describe block says `and useRequestQueuePreference is false`... so I'm not sure why this was true before. This works for now and will need to be modified soon once we remove that flag.